docs/src/configuration/routes/https.md

```diff
@@ -73,11 +73,6 @@ https://{default}/:
 
 ### `min_version`
 
-{{< note >}}
-This directive was put into place when Platform.sh supported older versions of TLS for customers.
-Currently only TLS v1.2 is supported. Support for TLS v1.3 will be added in the near future.
-{{< /note >}}
```

Comment:
Why remove?  It's worth telling people that you can't use <1.2 versions no matter what, especially for the security-conscious.

Reply:
First, I thought about just updating this `note` regarding TLS v1.3, as it is now supported, but right after this block, we talk about automatic rejections of connections with version lower than `min_version` and also say `Legal values are `TLSv1.2` and `TLSv1.3`.`
So it should be clear that no version lower than 1.2 can be used. No?

Reply:
Ah, fair point.  In that case, we should probably also update the example to use 1.3, so it's a non-default value.  And note that versions older than 1.2 will always be rejected, period.

Reply:
Updated in d4f41c8
WDYT?